examples/common_vmseries/README.md

```diff
@@ -513,7 +516,7 @@ Following properties are available:
 
   Please refer to [module documentation](../../modules/loadbalancer/README.md#frontend_ips) for available properties.
 
-  > [!NOTE] 
+  > [!NOTE]
```

Comment:
I was not touching load balancers in common design - I added only GWLB to common design. 
The same we have in other reference architecture examples e.g. https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/blob/5e8e2660b79d9adef45261abe9ba456dfc64bf6d/examples/dedicated_vmseries/variables.tf#L222

I propose to update LB settings in all examples in separate PR, ok ?

Here VS code automatically remove blank characters at the end of the line, so that's why you see changed file.

Reply:
I removed all changes from `examples/common_vmseries/README.md`